public_html/oneDimArr.js

```diff
@@ -0,0 +1,17 @@
+
+function arrayCreate(){
+    let n = parseInt(prompt("Please enter the size of the array:"));
+    let arr = [];
+    for(let i=0; i<n; i++){
+        arr[i] = parseInt(Math.floor(Math.random()*20));
```

Comment:
Random number generation is off by 1. It generates numbers between 0 and 19 instead of 1 and 20.